Index: test/css/hello.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test/css/hello.php	(revision 878b7162f9231ebe0c9ed27e920dbfef01684928)
+++ test/css/hello.php	(date 1579753093722)
@@ -2,4 +2,5 @@
 echo "you are";
 echo "yes";
 echo "hello";
-echo "no hello!!!!!";
\ No newline at end of file
+echo "no hello!!!!!";
+echo "yess hello";
\ No newline at end of file
