Index: test/css/hello.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test/css/hello.php	(revision 0aaf67141089a00e15f73d030f68bb39d9c5f9dd)
+++ test/css/hello.php	(date 1579752859404)
@@ -1,4 +1,5 @@
 <?php
 echo "you are";
 echo "yes";
-echo "hello";
\ No newline at end of file
+echo "hello";
+echo "no hello!!!!!";
\ No newline at end of file
